.vscode/settings.json

```diff
@@ -1,7 +0,0 @@
-{
-  "eslint.workingDirectories": [
```

Comment:
@garethfuller deleted this file from git cache as now it is .gitignored
But it would be good to use .editorconfig or similar.